self-api/src/main/java/com/selfxdsd/api/Organization.java

```diff
@@ -0,0 +1,64 @@
+package com.selfxdsd.api;
+
+import javax.json.JsonObject;
+
+/**
+ * Provider organization of which authenticated User has admin rights.
+ * @author criske
+ * @version $Id$
+ * @since 0.0.8
+ */
+public interface Organization {
+
+    /**
+     * Organization ID.
+     * @return String.
+     */
+    String organizationId();
+
+    /**
+     * Organization repos.
+     * @return OrganizationRepos.
+     */
+    OrganizationRepos repos();
```

Comment:
@criske Can you rename the interface ``OrganiztionRepos`` to simply ``Repos``? ``Repos`` is an interface which represents more repositories, but where these repositories are coming from is not important.

We will most likely have 2 main implementations of Repos: 1 for an Organization's repositories and 1 for a user's private repositories.